plugin.audio.kxmxpxtx.bandcamp/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.audio.kxmxpxtx.bandcamp" name="Bandcamp" version="0.4.1+matrix.1" provider-name="kxmxpxtx">
+<addon id="plugin.audio.kxmxpxtx.bandcamp" name="Bandcamp" version="0.5.0+matrix.1" provider-name="kxmxpxtx">
```

Comment:
Perhaps update the Provider name? As you are more or less taking over?

Reply:
@Virusmater stated that they are "not really interested anymore in this plugin" in the comment I linked earlier. So I think it's a good idea to take over as the provider for now. I can at least keep it running on future Kodi versions, as well as hopefully adding more features over time.

I'll add the changed provider to this PR.